addons/auth_totp/models/auth_totp.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from odoo import models
+
+
+class AuthTotpDevice(models.Model):
+
+    _name = "auth.totp.device"
+    _inherit = "res.users.apikeys"
+    _description = "Authentication Device"
+    _auto = False
```

Comment:
Maybe document that res.users.apikeys overrides `init` to set this up properly, otherwise it looks really weird (I almost asked why this was there without an `init` override before remembering there's one in the superclass).